supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiConfigurationAttributes.scala

```diff
@@ -287,25 +286,42 @@ object PipelinesApiConfigurationAttributes
     }
   }
 
-  def validateGcsPathToDockerImageCacheManifestFile(backendConfig: Config): ErrorOr[Option[ValidFullGcsPath]] = {
+  private [common] def validateGcsPathToDockerImageCacheManifestFile(backendConfig: Config): ErrorOr[Option[ValidFullGcsPath]] = {
     backendConfig.getAs[String]("docker-image-cache-manifest-file") match {
       case Some(gcsPath) => validateSingleGcsPath(gcsPath).map(Option.apply)
       case None => None.validNel
     }
   }
 
-  def validateGcsPathsToReferenceDiskManifestFiles(backendConfig: Config, backendName: String): ErrorOr[Option[List[ValidFullGcsPath]]] = {
-    backendConfig.getAs[List[String]]("reference-disk-localization-manifest-files") match {
-      case Some(gcsPaths) =>
-        logger.info(s"Reference disks feature for $backendName backend is configured with the following manifest files: ${gcsPaths.mkString(",")}.")
-        gcsPaths.map(validateSingleGcsPath)
-          .sequence
-          .contextualizeErrors("parse paths to file as valid GCS paths")
-          .map(Option.apply)
-
-      case None =>
-        logger.info(s"Reference disks feature for $backendName backend is not configured.")
-        None.validNel
+  /**
+    * Validate that the entries corresponding to "reference-disk-localization-manifests" in the specified
+    * backend are parseable as `ManifestFile`s.
+    */
+  private [common] def validateReferenceDiskManifestConfigs(backendConfig: Config, backendName: String): ErrorOr[Option[List[ManifestFile]]] = {
+    Try(backendConfig.getAs[List[Config]]("reference-disk-localization-manifests")) match {
+      case Failure(e) =>
+        e.getMessage.invalidNel
```

Comment:
ToL: This is fine according to our current coding practices, but someday I really hope we start tracking stack messages so we stop getting error reports like `Left(NonEmptyList("null"))`.

Reply:
Until that day I could provide a bit more context just in case the message is not helpful... 😉